controllers/gatekeeper_controller_test.go

```diff
@@ -557,32 +557,32 @@ func TestResources(t *testing.T) {
 	// test default resources
 	auditObj, err := util.GetManifestObject(AuditFile)
 	g.Expect(err).ToNot(HaveOccurred())
-	assertResources(g, auditObj, nil)
+	assertResources(g, auditObj, nil, true)
 	webhookObj, err := util.GetManifestObject(WebhookFile)
 	g.Expect(err).ToNot(HaveOccurred())
-	assertResources(g, webhookObj, nil)
+	assertResources(g, webhookObj, nil, false)
 
 	// test nil resources
 	err = crOverrides(gatekeeper, AuditFile, auditObj, namespace, false, false)
 	g.Expect(err).ToNot(HaveOccurred())
-	assertResources(g, auditObj, nil)
+	assertResources(g, auditObj, nil, true)
 	err = crOverrides(gatekeeper, WebhookFile, webhookObj, namespace, false, false)
 	g.Expect(err).ToNot(HaveOccurred())
-	assertResources(g, webhookObj, nil)
+	assertResources(g, webhookObj, nil, false)
 
 	// test resources override
 	gatekeeper.Spec.Audit = audit
 	err = crOverrides(gatekeeper, AuditFile, auditObj, namespace, false, false)
 	g.Expect(err).ToNot(HaveOccurred())
-	assertResources(g, auditObj, audit.Resources)
+	assertResources(g, auditObj, audit.Resources, true)
 
 	gatekeeper.Spec.Webhook = webhook
 	err = crOverrides(gatekeeper, WebhookFile, webhookObj, namespace, false, false)
 	g.Expect(err).ToNot(HaveOccurred())
-	assertResources(g, webhookObj, webhook.Resources)
+	assertResources(g, webhookObj, webhook.Resources, false)
 }
 
-func assertResources(g *WithT, obj *unstructured.Unstructured, expected *corev1.ResourceRequirements) {
+func assertResources(g *WithT, obj *unstructured.Unstructured, expected *corev1.ResourceRequirements, isAudit bool) {
```

Comment:
The `isAudit` boolean is unnecessary--you should just pass in the `expected` for all audit tests and leave the `assertResources` function as it is, right?